### PR TITLE
#8 Tags to redis

### LINKED ISSUE #211: tags.json --> save to redis!
edit csv-to-json.js to save the object to redis (instead of a .json file)

-------------

TODOs:

- [x] ~~edit csv-to-json.js file so that we're saving tags state to redis instead of to a .json file.~~
- [x] edit csv-to-json --> rename file to save-tags.js (read in .json, and save to redis)

### PR DESCRIPTION
**Pull request by mk4111**
Removes original .csv file, and the functionality to convert .csv to .json.
Keeps converted .json, and removes .json from .gitignore so it is pushed to GH
Makes changes from fs.readFile --> redis.HGET across multiple .js files.

Branches off of #209 
Closes #211

### CODE REVIEW

**Comment by mk4111** [comment]
@JMurphyWeb - would you mind having a look at this please? Thanks!

**Comment by nelsonic** [comment]
👀

File: tags/tags.json [hunk 1]
```diff
@@ -1 +1,797 @@
...
\ No newline at end of file
+[{
+    "id": 0,
```
Comment (nelsonic) [inline tags/tags.json]: @mk4111 are we expecting the EMF team to _manually_ edit this file?

**Comment by nelsonic** [comment]
@mk4111 code looks good. I would _eventually_ switch to `handleError` instead of `hoek.assert` to have fewer lines of code. but otherwise good to go. 👍

File: tags/tags.json (hunk 1 above)
Comment (mk4111) [inline tags/tags.json]: @nelsonic - yes - there's some instructions for the EMF team on how to update this file

**Comment by mk4111** [comment]
@nelsonic thank you! Will keep that in mind for the future

**Pull request merged**